app/models/event.rb

```diff
@@ -0,0 +1,41 @@
+class Event
+  DESCRIPTION_CUT = 140
+
+  def initialize(raw_event_details)
+    @_name = raw_event_details["title"]
+    @_date_time = raw_event_details["start_time"]
+    @_description = raw_event_details["description"]
+    @_venue_name = raw_event_details["venue_name"]
+    @_venue_url = raw_event_details["venue_url"]
+    @_venue_address = raw_event_details["venue_address"]
+  end
+
+  def name
```

Comment:
most of these methods could just be `attr_reader`'s if we use the typical instance variable syntax (not underscores)!